### PR TITLE
Refactor TFP call to just sigmoid()

### PR DESCRIPTION
**Pull request by Rocketknight1**
We're seeing [issues](https://huggingface.co/google/gemma-7b-it/discussions/71) caused by Tensorflow Probability + Keras 3 incompatibilities. Annoyingly, these are being thrown even when people are trying to use unrelated Torch models, as long as they have TF / TFP installed!

The underlying cause is that we import TFP in one of our pipeline files if it's present, and this file is run when `pipeline` is imported, even if the user is using an entirely different pipeline/framework. I removed the import entirely - it was only used to get the probabilities of a Bernoulli distribution given input logits, but this is just the sigmoid function, so I replaced the whole thing with a single call to `sigmoid()` instead.

### CODE REVIEW

**Review by amyeroberts — APPROVED** [review]
Much better 🧘  Thanks!
